### DISCUSSION TITLE
the 17th Marcel Grossmann Meeting

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#216] [General]
[Seventeenth Marcel Grossmann Meeting](https://indico.icranet.org/event/8/)
"Since 1975, the Marcel Grossmann  Meetings (on Recent Developments in Theoretical and Experimental General Relativity, Gravitation, and Relativistic Field Theories) have been organized in order to provide opportunities for discussing recent advances in gravitation, general relativity and relativistic field theories, emphasizing mathematical foundations, physical predictions and experimental tests. The objective of these meetings is to elicit exchange among scientists that may deepen our understanding of spacetime structures as well as to review the status of ongoing experiments aimed at testing Einstein's theory of gravitation either from the ground or from space."

**Comment by RedshiftDrift** [comment]
Alessandro Spallicci strongly insisted that I participate in [the 17th MG meeting](https://indico.icranet.org/event/8/).  Considering what I think about the space-expanding interpretation of GR, it seems paradoxal that I would attend a meeting called "Marcel Grossmann", the mathematician who helped Einstein by introducing space-time mathematics.  Don't get me wrong, I admire Grossmann's mathematical skills, but his physics is just as "abominable" as Lemaître's 😉 

  Then I read Spallicci's description of the session *"Extended theories of electromagnetism and their impact on laboratory experiments and astrophysical observations"* (second last at the bottom of the [webpage](https://indico.icranet.org/event/8/program)).  This description particularly attracted my attention:
  *"Despite the detection of neutrinos, cosmic rays and recently gravitational waves, astronomy is largely based on electromagnetic signals, still interpreted with 19th century Maxwellian linear electromagnetism. But the latter might not be the right tool to interpret any electromagnetic phenomenon."*

  That explains the sad state of astronomy/cosmology, they are two centuries behind!  In the 160 years since Maxwell, there have been discoveries that go well beyond his equations: blackbody radiation, the photoelectric effect, stimulated emission, the quantization of E-B fields, atomic physics and spectroscopic features, the laser, optical quantum states and a myriad of effects such as laser manipulation of atoms and charged particles, harmonic generation, optical cooling to nanokelvin temperatures, dissipative processes and atomic trapping.

  I registered and will give a talk to the *19th century* participants.
  It's not too late to register more ACG members, @Francois-Zinserling ?

**Reply by Francois-Zinserling** [reply]
Wait, what? Why me? I am highly **un**qualified for this and there are other much more serious researchers here. :)

**Reply by RedshiftDrift** [reply]
You are too modest...

**Reply by RedshiftDrift** [reply]
Speaking of **un**qualified for this 😉 my talk has been scheduled for July 8th at 18:05CEST.
["Dispersive interactions between photons and electrons, an application to cosmology"](https://indico.icranet.org/event/8/contributions/1535/)

**Reply by Pierre314159** [reply]
Go get’em, Louis!

**Reply by ExpEarth** [reply]
Exciting!  Can we watch your talk online?  If so, is that Eastern Standard Time?

**Reply by Francois-Zinserling** [reply]
Send invites to the talk!

**Reply by RedshiftDrift** [reply]
@ExpEarth CEST is "Central European Summer Time".
I don't think the conference will be accessible online, but... see below...

**Comment by RedshiftDrift** [comment]
As suggested by Roy, I will have a rehearsal of my MG17 talk on Zoom for the benefit of AGC members.
  There will be time after the talk for feedback and questions.
  The 90-minute Zoom session is scheduled for Friday June 28 at [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20240628T140000&p1=1440&p2=188)

If you are interested in attending the Zoom session, send an email to my adress [here](https://cosmology.info/org/acg-organization.html#contacts) and I will send you the Zoom link.

**Reply by Francois-Zinserling** [reply]
Returned from work too late and missed the prep talk :(
I hope all went well, and I hope there is a recording?

**Reply by HanDeBruijn** [reply]
A few remarks are in place, I think.

Because it's a Zoom session, I would have expected a more user friendly introduction.
For example, "Roy, are you there? Would you please raise your hand! Okay."
"Pierre, are you there? Would you please raise your hand! Okay." And so on.
Some people could be recognized by display of their names, others could not.
The title of my subscreen said "Thuis PC" and I didn't know how to change that on the fly.

The talk itself was not very deviant from what is common with such "scientific" talks.
Speaking for myself, I felt there was little understanding and spirit among the participants.
A bit more pep talk, performance, _surely you’re joking mr. Feynman_, would that be a good idea?

Wish you the best anyway, Louis, with the final presentation.

**Reply by Pierre314159** [reply]
Oh, so “This PC” was you? Nice to met you, Han! 😜

**Reply by RedshiftDrift** [reply]
>  I would have expected a more user friendly introduction

Many people were late 😞 and missed the friendly introduction.  Usually, people introduce themselves when they ask questions or give a comment.

Unfortunately, I am not able to give a lively talk *à la* Feynman... my brain is not wired for social events.  But after the talk I got feedback from a few people by email. I know at least two people understood what I said very well.

**Comment by HanDeBruijn** [comment]
Among the [Public lectures](https://indico.icranet.org/event/8/page/51-public-lectures) at the Meeting there is an interesting one by [Wendy Freedman](https://en.wikipedia.org/wiki/Wendy_Freedman).
From her video titled [Our Expanding Universe: Wendy L. Freedman 2023 Ryerson Lecture](https://www.youtube.com/watch?v=7IqCGN5Z3lg) I've made two screenshots.

![wendy_red](https://github.com/a-cosmology-group/acg/assets/135433222/fe3d6e51-e767-4bef-a004-cbaff3084d9d)
The Hubble constant for Red giants is in agreement with the best value (70 km/s/Mpc) as [found](https://github.com/orgs/a-cosmology-group/discussions/225#discussioncomment-9909045) by @mikehelland for a static universe.

![wendy](https://github.com/a-cosmology-group/acg/assets/135433222/657992f3-a1b2-40f0-bdab-b43ac60b1ee8)

**Reply by mikehelland** [reply]
Also note that Freedman's value as been about 70 for years. Here's a chart with measurements citing hers from 2020:

![image](https://github.com/a-cosmology-group/acg/assets/19311107/eb3813c3-ed2b-401d-9749-65cec18b810e)

It's a minor point, but I've seen at referred to as a new measurement (2023 or 2024) though it's been around for a while. Note though, it still uses LCDM with $\Omega_m$ about 0.3.

**Comment by mikehelland** [comment]
Have a great talk today! (Assuming you haven't already.)

PS, if you see Roy Kerr around (he's on the list), please ask him where the "GR is about forces, not geometry" Einstein quote comes from. I haven't been able to track that down, and others have questioned my citing it.

**Reply by RedshiftDrift** [reply]
Hi @mikehelland, because of my illness in May, I was forced to participate virtually via Zoom (I am *not* in Italy 😞)  The session chairperson knows everything about Lagrangians in four dimensional space, but unfortunately can't handle the reality of projection screens and loudspeakers (a stereotypical theorist).  So far I have only seen the chairperson and another participant.  I am at the mercy of the laptop that virtually allows me to see the wall of a conference room...

I won't be able to have a reasonable discussion with anyone, except perhaps with the few people attending the session for the discussion session on Friday... if they can hear me.
